deploy/tests/test_generate_production_ini.py

```diff
@@ -14,6 +14,13 @@
     template_environment_names,
 )
 
+
+# TODO: Maybe this should move to env_utils? If not, at least to a non-test file.
+#       Then again, if we used the "single parameterized ini file" we could side-step that. -kmp 3-Apr-2020
+
+FOURFRONT_DEPLOY_NAMES = ['hotseat', 'mastertest', 'webdev', 'webprod', 'webprod2']
```

Comment:
Shouldn't this include blue and green?

Reply:
Yes. Although I don't think these tests are even running by default. I'm changing `pytest.ini`, too. Please double-check that.